app/hooks/useAmplitude.ts

```diff
@@ -50,7 +51,7 @@ export function useAmplitude() {
 
   const trackSkjemaSteg = useCallback(
     ({ periode: { id, rapporteringstype }, stegnavn, steg, endring = false }: ISkjemaSteg) =>
-      trackEvent("skjemasteg fullført", {
+      trackEvent("skjema steg fullført", {
```

Comment:
@abdifatah-bashi , dette følger taksonomien fra researchops. Hvilke konsekvenser får det å endre dette for allerede eksisterende data i amplitude?